docs/cookbook/migrations/homebrew.md

```diff
@@ -0,0 +1,178 @@
+
+```
+% flox search boost
+boost     Collection of C++ libraries
+booster   Fast and secure initramfs generator
+xgboost   Scalable, Portable and Distributed Gradient Boosting (GBDT, GBRT or GBM) Library
+boost155  Collection of C++ libraries
+boost159  Collection of C++ libraries
+boost15x  Collection of C++ libraries
+boost160  Collection of C++ libraries
+boost165  Collection of C++ libraries
+boost166  Collection of C++ libraries
+boost167  Collection of C++ libraries
+
+Showing 10 of 120 results. Use `flox search boost --all` to see the full list.
+
+Use 'flox show <package>' to see available versions
+```
+
+The first one on the list is the correct Flox package to install, and it has the same name as the Homebrew package. This will often be the case, but not always.
+
+### Install your first package
+To install your first package, use `flox install`:
+
+```
+% flox install boost
+```
+
+The first time you install a package, Flox will ask you whether you want to create a default environment. If you choose to do so, Flox will then ask you whether you want to configure your dotfiles to automatically activate the new default environment.
+
+```
+[show the flow]
+```
+
+If you do not choose for this to be automated at the time of your first package installation, you can [follow these instructions]() to add Flox to your dotfiles manually.
+
+### Verifying configuration
+Exit your active shell and create a new one, causing the dotfile changes to take effect. The first time this happens, you may experience a short delay while your default environment is materialized.
+
+Once the shell is available, verify that your default environment is active by running `flox` with no arguments:
+
+```
+% flox
+flox version 1.3.6
+
+Usage: flox OPTIONS (init|activate|search|install|...) [--help]
+
+Use 'flox --help' for full list of commands and more information
+
+✨ Active environments:
+  default           /Users/floxfan
+```
+
+If you see `default` listed amongst the active environments, your dotfiles have been correctly modified. The default environment will be active whenever you log in.
+
+Verify that your package has been installed using `flox list`:
+
+```
+% flox list
+boost: boost (boost@1.81.0)
+```
+
+## Create project environments
+As you continue to migrate packages from Homebrew to Flox, you may find that you don’t need them all in your default environment.
+
+The default environment is intended for packages that should be available to the user across all of the contexts where they work. It is commonly used for general utilities like `gh`, `gnused`, and `curl` that apply to many situations.
+
+Packages that are required for specific contexts (e.g., different projects in a monorepo, different repos, different customer deployments, etc.) are often installed into project environments which are then activated in addition to the default environment.
```

Comment:
We don't talk about "project environments" as a concept anywhere, so maybe consider dropping that as an adjective. Instead, could say just "environments" and in the section title "Create environments for projects".

The concept page for environments uses the terms "path" and "centrally managed" to talk about the use cases for environments, and I think in this case you're referencing the "path" use case. My goal is to reduce the number of terms a user has to understand.